pkg/email/templates/trb_advice_letter_submitted.gohtml

```diff
@@ -0,0 +1,25 @@
+{{template "easi_header.gohtml"}}
+
+<p>The Technical Review Board (TRB) has compiled an advice letter for {{.RequestName}}. Use the link below to view recommendations from the TRB as well as a summary of the initial support request.</p>
+
+<p><a href="{{.TRBAdviceLetterLink}}" style="font-weight: bold">View the Advice Letter</a></p>
+
+<p>Any further communication or follow-up sessions may be organized via email with the TRB or via a new request in EASi.</p>
+
+<p><strong>Request summary</strong></p>
+<ul style="padding-left: 0;">
+<li style="list-style-type: none;">Submission date: {{.SubmissionDate}}</li>
+<li style="list-style-type: none;">Requester: {{.RequesterName}}</li>
+<li style="list-style-type: none;">Component: {{.Component}}</li>
```

Comment:
Perhaps having sometihng like "None Selected" instead of "" in cases where this is `nil` would be nice? Looking at the main screenshot, looks like you can just get `Component: `, which might give off the implication that something went wrong, rather than it not having been selected?